salt/master.py

```diff
@@ -1909,6 +1910,18 @@ def publish(self, clear_load):
         payload = self._prep_pub(minions, jid, clear_load, extra, missing)
 
         # Send it!
+        if self.opts[u'enable_ssh_minions'] is True and isinstance(clear_load[u'tgt'], six.string_types):
+            # The isinstances makes sure that syndics work
+            log.debug('Use SSHClient for rostered minions')
+            ssh = salt.client.ssh.client.SSHClient()
+            ssh_minions = ssh._prep_ssh(**clear_load).targets.keys()
+            if ssh_minions:
+                minions.extend(ssh_minions)
+                def wrap_ssh(**kwargs):
+                    salt.utils.process.daemonize(False)
+                    ssh.cmd(**kwargs)
+                salt.utils.process.SignalHandlingMultiprocessingProcess(target=wrap_ssh, kwargs=clear_load).start()
```

Comment:
This process is still becoming a zombie, even if it is daemonized, whatever runs the ssh connection is causing it to be zombied, possibly because of the generator that is created in `salt.client.ssh`, but if this is switched to threading the whole problem goes away.

@cachedout can you take a look.  Tom and I looked for a couple hours on Friday but couldn't figure out where to fix this.  We have the suspicion that the same thing also happens when running `salt-ssh` from the commandline, just that the zombie is cleaned up in a matter of milliseconds by the shell.

Thanks,
Daniel